3_navigation.ipynb

```diff
@@ -205,6 +392,57 @@
     "以下のコマンドでx=1メートル、y=0.5メートル、theta=90度の位置姿勢に移動できます。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Send a goal\n",
+    "To send a goal to the robot, execute the following command and observe the movement of the robot on the simulator.\n",
```

Comment:
"To send a goal to the robot, execute the following command and observe the movement of the robot in the simulator.\n",